src/components/notification/NotificationsWrapper.js

```diff
@@ -1,12 +1,17 @@
 // @flow
 import * as React from 'react';
 
+import FocusTrap from '../focus-trap';
 import Portal from '../portal';
 
 type Props = {
     children?: React.Node,
 };
 
-const NotificationsWrapper = ({ children }: Props) => <Portal className="notifications-wrapper">{children}</Portal>;
+const NotificationsWrapper = ({ children }: Props) => (
+    <Portal className="notifications-wrapper" aria-live="polite">
+        {children ? <FocusTrap>{children}</FocusTrap> : null}
```

Comment:
Children should be required, no? Otherwise what is this component rendering?

Reply:
It was already nullable, so seems like we're just maintaining the existing API?

Reply:
maybe should fix the flow type (though unsure what other component flow might require fixing)

Reply:
For now goal is to maintain existing API. I agree that children should be required.